defaults/test_default.yaml

```diff
@@ -4,7 +4,7 @@ ip_ssh_connections: 'private'
 
 mgmt_port: 10090
 scylla_repo_m: 'http://repositories.scylladb.com/scylla/repo/qa-test/centos/scylladb-2019.1.repo'
-scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo'
+scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/branch-2.0/24/scylla-manager.repo'
```

Comment:
We should have a promoted manager release now, so let’s use the official or latest of if we want it to progress but not a fixed build.